nodeenv.py

```diff
@@ -577,17 +577,24 @@ def tarfile_open(*args, **kwargs):
         tf.close()
 
 
+def _download_node_file(node_url, n_attempt=3):
+    """Do multiple attempts to avoid incomplete data in case of unstable network"""
+    while n_attempt > 0:
+        try:
+            return io.BytesIO(urlopen(node_url).read())
+        except IncompleteRead as e:
+            print('Incomplete read while reading from {} - {}'.format(node_url, e))
```

Comment:
Suggest to use logger instead of print here (as it was before this change).

Reply:
Fxed. Good catch :+1: